dc-infrastructure.tf

```diff
@@ -46,9 +46,10 @@ module "bamboo" {
     mem          = var.bamboo_mem
     min_heap     = var.bamboo_min_heap
     max_heap     = var.bamboo_max_heap
-    license      = var.bamboo_license
   }
 
+  license = var.bamboo_license
```

Comment:
Out of curiosity, What is the purpose of `bamboo_configuration` block?

Reply:
This is for grouping the configuration settings. We had a discussion on this. Originally this is implemented to make the code more clear and organised. However, there is a point that we may better follow the common practise and use separated variables as other modules in market doing the same. However, we don't expose internal modules (module folder and products) this method does not violate the common practise but still this is an option for refactor in future.